tests/storage/snapshots/test_snapshots.py

```diff
@@ -110,34 +110,34 @@ class TestRestoreSnapshots:
                 id="test_restore_all_snapshots",
             ),
         ],
-        indirect=["cirros_vm_name", "snapshots_with_content"],
+        indirect=["rhel_vm_name", "snapshot_with_content"],
     )
     def test_restore_snapshots(
         self,
-        cirros_vm_for_snapshot,
-        snapshots_with_content,
+        rhel_vm_for_snapshot,
+        snapshot_with_content,
         expected_results,
         snapshots_to_restore_idx,
     ):
         for idx in range(len(snapshots_to_restore_idx)):
             snap_idx = snapshots_to_restore_idx[idx]
             with VirtualMachineRestore(
                 name=f"restore-snapshot-{snap_idx}",
-                namespace=cirros_vm_for_snapshot.namespace,
-                vm_name=cirros_vm_for_snapshot.name,
-                snapshot_name=snapshots_with_content[snap_idx].name,
+                namespace=rhel_vm_for_snapshot.namespace,
+                vm_name=rhel_vm_for_snapshot.name,
+                snapshot_name=snapshot_with_content[snap_idx].name,
             ) as vm_restore:
                 vm_restore.wait_restore_done()
-                cirros_vm_for_snapshot.start(wait=True)
-                run_command_on_cirros_vm_and_check_output(
-                    vm=cirros_vm_for_snapshot,
+                rhel_vm_for_snapshot.start(wait=True)
```

Comment:
Please replace with `running_vm()` in all places (unless there's a good reason not to)
```suggestion
                running_vm(vm=rhel_vm_for_snapshot)
```